### DISCUSSION TITLE
Swup on GitHub Pages

### DISCUSSION THREAD

**Discussion by ArsenPreon** [swup/swup#550] [Help] [answered]
I downloaded setup js and when I was working with links on a local server, swap worked and switched pages without reloading, but when I posted the site on github pages, I had to change the link attributes `<a href="/index.html ">` `<a href="https://ArsenPreon .github.io/SkyPlazaHotel/number.html ">` the swup has stopped working can anyone know what can be done?

**Comment by daun** [comment, marked as answer]
While domain names are technically case-insensitive, html attributes like `href` are case-sensitive, though. That's why swup isn't picking up your links. If you stick to the lowercase version of your subdomain for all links, everything should work as expected.

**Reply by ArsenPreon** [reply]
thank you, this is what I wanted to hear, I made a translation into English in the previous one, but no one answered me and I decided to recreate the discussion
